test/inductor/test_cpu_select_algorithm.py

```diff
@@ -2936,8 +2936,10 @@ def forward(self, x):
 
         x = torch.randn(batch_size, in_features).to(dtype=dtype)
         mod = M().to(dtype=dtype).eval()
-        self.common(mod, (x))
-        _, code = run_and_get_cpp_code(mod, x)
+        with verify(dtype) as (atol, rtol):
+            self.common(mod, (x,), atol=atol, rtol=rtol)
+        m = torch.compile(mod)
```

Comment:
instead of using `self.common`, I think you can use compiled m directly to check the accuracy.

Reply:
Thanks for your comments. Revised.